### ISSUE TITLE
loss is nan

### ISSUE THREAD

**Issue by zhouxia01** [prs-eth/Dynamic-LiDAR-Resimulation#1]
After starting training, I found that many loss was nan. I don’t know if this was expected.

**Comment by hanfeng-wu** [comment]
Did you use float32 or float16?

**Comment by hanfeng-wu** [comment]
No, there should not be any NaN losses, which losses are NaN?

**Comment by zhouxia01** [comment]
It was caused by the installation package version. The problem has been solved. However, there are two problems with the model: (1) There is a difference between the MAE of the model and the value in the paper (40 vs 30.8); (2) There is noise in the blank space. Is it because the parameters are wrong? The parameters are not modified and the data is downloaded directly.

**Comment by ShengyuH** [comment]
hi,

Thanks for your interest in our work. Did you set batch size to 4096 as reported in our paper? We re-factor the codebase for this release and are now investigating this issue, and will come back with the full script for reproduction along with the checkpoints. 

Best,
Shengyu

**Comment by hanfeng-wu** [comment]
Hi,

We just uploaded the pretrained weights for our dataset and reproduction scripts, you can check it out
